packages/grafana-ui/src/types/datasource.ts

```diff
@@ -508,6 +503,7 @@ export interface DataSourceSettings<T extends DataSourceJsonData = DataSourceJso
   secureJsonFields?: KeyValue<boolean>;
   readOnly: boolean;
   withCredentials: boolean;
+  version?: number;
```

Comment:
After the change to `DataSourcePluginOptionsEditorProps` this popped up. Version is something that you can set (but probably should be done automatically) and some datasources would set it manually. They were typed as `any` so it wasn't an error before.

Reply:
good catch!